test/spec/sparkpost.spec.js

```diff
@@ -248,14 +255,41 @@ describe('SparkPost Library', function() {
         done();
       });
     });
+
+    it('should return a parsed JSON object', function(done) {
+      nock('https://test.sparkpost.com')
+        .get('/test')
+        .reply(200, '{ "ok": true }');
```

Comment:
If you give nock and object to return, it will do stuff on it's side. I wanted to make sure that the response was a string and was being manipulated by the request lib.